cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/SystemArtifactLoader.java

```diff
@@ -42,10 +48,12 @@ public Service get() {
           protected void doStart() {
             try {
               artifactRepository.addSystemArtifacts();
+              LOG.info("Successfully added system artifacts");
```

Comment:
this can be a misleading message as this can complete without adding anything.  It already logs when a system artifact is added and when an artifact is skipped, don't think we need an additional message.